Lesson26/app/deniz/script.js

```diff
@@ -0,0 +1,52 @@
+// Register on Unsplash to get your own API key
+const API_KEY = '3HO6COGJ_0Qsi2DxSVvlppxQBAHZj4jCPBlARAusM3g';
+
+// TODO: Add a new button in HTML to crear the screen
+
+const button = document.getElementById('fetch-button');
+button.addEventListener('click', function () {
+  let picCount = parseInt(document.getElementById("pic-count").value) || 10;
+  
+  if (picCount <= 0) {
+    document.getElementById("error-message").textContent = "Please enter a postive number.";
+    return;
+  }
+  // TODO: Add an input in HTML (e.g. slider) to select how many new picutres should be fetched. Depending on the user input, the count value in the request should be changed to value selected by user
+  fetch(`https://api.unsplash.com/photos/random?count=${picCount}&client_id=${API_KEY}`)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error("Network response was not ok " + response.statusText);
+      }
+      return response.json();
+    })
+    .then(data => {
+      if (data.errors) {
+        throw new Error(data.errors[0]); // burada neden (data.errors[0]) kullaniyoruz ?
```

Comment:
When an error occurs, it could be that there are more than one error so that the errors are returned in an array format. Accessing the first element of the array would allow to access the first error. You could try to check if there are more than one error, concatenate them into one string and add that string to the Error but it's not very much necessary.